prime-router/docs/proposals/0019-server-server-authentication-updates.md

```diff
@@ -0,0 +1,36 @@
+## Background
```

Comment:
Do we need this file considering we have 0002-authentication.md?

Reply:
My thought was if the documentation regarding the feature is updated to match how the feature actually is, then we don't need an extra document every time we update that documentation?

Reply:
Yep I'm good with that!

Reply:
Deleted